pre_commit_hooks/pretty_format_json.py

```diff
@@ -120,7 +120,10 @@ def main(argv=None):  # type: (Optional[Sequence[str]]) -> int
             )
 
             if contents != pretty_contents:
-                print('File {} is not pretty-formatted'.format(json_file))
+                print(
+                    'File {} is not pretty-formatted'.format(json_file),
+                    file=sys.stderr,
+                )
```

Comment:
you'll want a `sys.stderr.flush()` after this as it won't line up always otherwise when piped